src/main/java/com/solr/clientwrapper/rest/KafkaResource.java

```diff
@@ -0,0 +1,46 @@
+package com.solr.clientwrapper.rest;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.solr.clientwrapper.domain.service.KafkaSender;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.security.SecurityRequirement;
+
+@RestController
+@RequestMapping("/kafka")
+public class KafkaResource {
+
+	private final Logger log = LoggerFactory.getLogger(KafkaResource.class);
+
+	@Autowired
+	private KafkaSender sender;
+
+	@PostMapping("/Add/{AddToQueue}")
```

Comment:
Best practice is to use **_hyphenated words_** instead of SnakeCase or camelCase in _APIs URIs_

Reply:
add-to-queue

Reply:
done